android_studio_project/praca_inz_03/app/src/main/java/com/example/praca_inz_03/DodajNowyTerminLekarz.java

```diff
@@ -0,0 +1,130 @@
+        final Calendar kalendarz=Calendar.getInstance();
+
+
+                DatePickerDialog.OnDateSetListener wyborDaty=new DatePickerDialog.OnDateSetListener() {
+            @Override
+            public void onDateSet(DatePicker view, int year, int month, int dayOfMonth) {
+                kalendarz.set(Calendar.YEAR,year);
+                kalendarz.set(Calendar.MONTH,month);
+                kalendarz.set(Calendar.DAY_OF_MONTH,dayOfMonth);
+
+                TimePickerDialog.OnTimeSetListener wyborGodziny=new TimePickerDialog.OnTimeSetListener() {
+                    @Override
+                    public void onTimeSet(TimePicker view, int hourOfDay, int minute) {
+                        kalendarz.set(Calendar.HOUR_OF_DAY,hourOfDay);
+                        kalendarz.set(Calendar.MINUTE,minute);
+
+                        SimpleDateFormat formatDaty=new SimpleDateFormat("yyyy-MM-dd HH:mm:00");
+
+                        date_time_in.setText(formatDaty.format(kalendarz.getTime()));
+                    }
+                };
+
+                new TimePickerDialog(DodajNowyTerminLekarz.this,wyborGodziny,kalendarz.get(Calendar.HOUR_OF_DAY),kalendarz.get(Calendar.MINUTE),false).show();
+            }
+        };
+
+        new DatePickerDialog(DodajNowyTerminLekarz.this,wyborDaty,kalendarz.get(Calendar.YEAR),kalendarz.get(Calendar.MONTH),kalendarz.get(Calendar.DAY_OF_MONTH)).show();
+
+    }
+    private void pokazWyborWizytyStop(final EditText date_time_in) {
+        final Calendar kalendarz=Calendar.getInstance();
+        DatePickerDialog.OnDateSetListener wyborDaty=new DatePickerDialog.OnDateSetListener() {
+            @Override
+            public void onDateSet(DatePicker view, int year, int month, int dayOfMonth) {
+                kalendarz.set(Calendar.YEAR,year);
+                kalendarz.set(Calendar.MONTH,month);
+                kalendarz.set(Calendar.DAY_OF_MONTH,dayOfMonth);
+
+                TimePickerDialog.OnTimeSetListener wyborGodziny=new TimePickerDialog.OnTimeSetListener() {
+                    @Override
+                    public void onTimeSet(TimePicker view, int hourOfDay, int minute) {
+                        kalendarz.set(Calendar.HOUR_OF_DAY,hourOfDay);
+                        kalendarz.set(Calendar.MINUTE,minute);
+
+                        SimpleDateFormat formatDaty=new SimpleDateFormat("yyyy-MM-dd HH:mm:00");
+
+                        date_time_in.setText(formatDaty.format(kalendarz.getTime()));
+                    }
+                };
+
+                new TimePickerDialog(DodajNowyTerminLekarz.this,wyborGodziny,kalendarz.get(Calendar.HOUR_OF_DAY),kalendarz.get(Calendar.MINUTE),false).show();
+            }
+        };
+
+        new DatePickerDialog(DodajNowyTerminLekarz.this,wyborDaty,kalendarz.get(Calendar.YEAR),kalendarz.get(Calendar.MONTH),kalendarz.get(Calendar.DAY_OF_MONTH)).show();
+
+    }
+
+    public void DodajTermin(View view) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.